src/views/v2/rsvp/ari/sidebar/quantity.php

```diff
@@ -13,7 +15,15 @@
 ?>
 <div class="tribe-tickets__rsvp-ar-quantity">
 	<span class="tribe-common-h7">
-		<?php esc_html_e( 'Total Guests', 'event-tickets' ); ?>
+		<?php
+		echo wp_kses_post(
```

Comment:
Why kses?

Reply:
I thought they may want use a bit of html styling here (`strong` or something ,for some word).

Reply:
I wonder if the html part would be template-specific, where someone wanting to add `<strong>` would come in here and make that change themselves, in which case they could add the `kses`? I don't think one could add html as a replacement for `tribe_get_guest_label_plural()`?